### PR TITLE
Add a current_item property

### PR DESCRIPTION
**Pull request by noirbizarre**
I added a `current_item` property on NavigationBar to simplify the writing of parent templates.

It allows writing:

``` html+jinja
<h2>{{ nav.top.current_item.label }}</h2>
```

### CODE REVIEW

**Comment by shonenada** [comment]
@noirbizarre Thanks!

File: flask_navigation/navbar.py [hunk 1]
```diff
@@ -35,3 +35,11 @@ def alias_item(self, alias):
...
         ident = self.alias[alias]
         return self.items[ident]
+
+    @property
+    def current_item(self):
+        '''Get the current active navigation Item if any'''
+        for item in self:
+            if item.is_active:
+                return item
+        return None
```
Comment (tonyseek) [inline flask_navigation/navbar.py]: The `return None` could be removed :-)

**Comment by noirbizarre** [comment]
I rebased the pull-request, removed the `return None` and also handle nested items.

Is it possible to release this ?

Thanks!!

**Comment by tonyseek** [comment]
@noirbizarre Thank you very much :-)

I will release a new version tomorrow.

**Comment by noirbizarre** [comment]
Thanks! You're the best!

**Pull request merged**